Authentication events policy docs for user flow invocation (#10224)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/authenticationsourcefilter.md

---
title: "authenticationSourceFilter resource type"
description: "Filter based on the source of the authentication which is used to determine whether the action is executed or not."
author: "jkdouglas"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# authenticationSourceFilter resource type

Namespace: microsoft.graph

> Filter based on the source of the authentication which is used to determine whether the action is executed or not.

Reviewer: ```suggestion
Filter based on the source of the authentication that is used to determine whether the action is executed.
```

## Properties

|Property|Type|Description|
|:---|:---|:---|

> |includeApplications|String collection|Applications to include. These applications will trigger the associated action when used as the client application in the authentication flow.|

Reviewer: ```suggestion
|includeApplications|String collection|Applications to include. These applications trigger the associated action when used as the client application in the authentication flow.|
```
